sdk/src/client/api/block_builder/transaction_builder/transition.rs

```diff
@@ -172,7 +287,80 @@ impl TransactionBuilder {
             Output::Account(account_input) => self.transition_account_input(account_input, input.output_id()),
             Output::Foundry(foundry_input) => self.transition_foundry_input(foundry_input, input.output_id()),
             Output::Nft(nft_input) => self.transition_nft_input(nft_input, input.output_id()),
+            Output::Basic(basic_output) => self.transition_implicit_account_input(basic_output, input.output_id()),
             _ => Ok(None),
         }
     }
 }
+
+#[derive(Debug, Clone, Eq, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub enum AccountChange {
```

Comment:
What about other changes like adding a metadata feature? Would we need an enum variant for all of that or can we still do that different?

Reply:
I think that's definitely possible, I just added only the stuff we currently use

Reply:
What do you mean with it's possible? Would we need to add them to the enum variant or can you transition an output and do the changes without using this enum?

Reply:
You can do it manually still by adding the output, but that output cannot be changed to add mana or anything so sometimes you need an additional remainder. We could make it possible to do it entirely in the builder using this `Transitions` type

Reply:
Hm and what if you want to do multiple changes to the output and still get mana remainder added to it?

Reply:
Currently that would not be possible. Do you think it's worth adding this kind of thing?

Reply:
Not sure, but looks like this enum would end up being another account output builder which doesn't really seem to be a good solution to me

Reply:
Well, we can always change it if there's a need in the future 🤷‍♀️